runtime/src/iree/hal/drivers/cuda/dynamic_symbols.c

```diff
@@ -210,3 +206,12 @@ void iree_hal_cuda_dynamic_symbols_deinitialize(
 
   IREE_TRACE_ZONE_END(z0);
 }
+
+iree_status_t make_missing_nccl_status() {
```

Comment:
let's re-inline this to the two locations - not worth exposing the symbol for maintenance reasons (less cross-dependencies make it easier to refactor code) - if anything we should reduce the places that check for the library/version to a single location instead

Reply:
Done.

Reply:
Could you check that this is resolved?